dependency-convergence/src/main/java/cloudbomcommitcheck/RecentCommitCheck.java

```diff
@@ -132,7 +144,7 @@ public static int execute(String commitMessage) throws ParseException, MavenRepo
     return 0;
   }
 
-  private static void printOutput(int exitCode) {
+  private void printOutput(int exitCode) {
```

Comment:
why is this no longer static?

Reply:
Jeff mentioned that I should make everything instanced instead of static for more maintainable code - I decided it's prudent for it to remain static instead of passing in a RecentCommitCheck object because I thought the design would be cleaner